### DISCUSSION TITLE
POSTGIS AND OFFLINE ISSUE ON DUPLICATE FEATURE

### DISCUSSION THREAD

**Discussion by aliasatp** [opengisch/QField#3401] [Q&A]
Hi All, I have a project based on postgis layers. In Qfield I work in offline mode so my layers have been packed in gpkg format. When I duplicate a feature I have no problem in Qfield as expected but on cloud side the sync action is not applied. I think it could be due to PK field conflict. Problably the PK in gpkg is managed as ordinary field (not unique) and when the feature is sent back to POSTGIS it is an error for the DB. Any idea?

**Comment by suricactus** [comment]
Yes, I would also think this is the problem. It is definitely something that needs to be fixed, but I am quite busy next few weeks. Will ping you back when it is done.
